images/installer-origin-release/Dockerfile.ci

```diff
@@ -1,7 +1,6 @@
 # This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
-# FIXME: Move this image to an openshift repo?
 
-FROM  quay.io/coreos/origin-release-oc:bazel
+FROM  openshift/origin-release:bazel
```

Comment:
I'm fine with this as you have it.  I've been arguing for registry prefixes recently, but this is a CI-only Dockerfile (with a `.ci` suffix), and there I think the lack of a prefix makes more sense.